src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,8 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
```

Comment:
Nice! Now we just need to make this variable a field(class level variable) of an `AuthenticationService` class to not create a new `UserService` each time we call `login()` method, but to reuse a single instance each time

Reply:
If i understood right, i need to move the "UserService userService = new UserService();" out of the method to the class and make it private as well?

Reply:
Yep, next time if you have any questions do not be shy to tag me in your private chat on the platform and ask them there because it is hard to see it here :)